src/libraries/System.Formats.Tar/tests/TarTestsBase.cs

```diff
@@ -409,7 +409,8 @@ protected static void AssertEntryModeFromFileSystemEquals(TarEntry entry, UnixFi
 
         protected static void AssertFileModeEquals(string path, UnixFileMode mode)
         {
-            if (!PlatformDetection.IsWindows)
+            if (!PlatformDetection.IsWindows &&
+                !PlatformDetection.IsAndroid) // Android may change the requested permissions.)
```

Comment:
Do you have any docs/links/etc or more information on exactly what/why Android is changing the requested permissions?